ghost/core/core/server/services/mega/post-email-serializer.js

```diff
@@ -107,6 +108,23 @@ const PostEmailSerializer = {
         return signupUrl.href;
     },
 
+    /**
+     * createUserLinks
+     *
+     * Generate personalised links for each user
+     *
+     * @param {string} memberUuid member uuid
+     * @param {Object} email
+     */
+    createUserLinks(email, memberUuid) {
+        const result = {...email};
+
+        result.html = feedbackButtons.generateLinks(result.post.id, memberUuid, result.html);
+        result.plaintext = htmlToPlaintext.email(result.html);
```

Comment:
Is it necessary that we do this here?

Reply:
I'd suggest keeping it :)  We added new links to Html and I think it's better to update the plain version